modules/peers.js

```diff
@@ -1,35 +1,32 @@
 'use strict';
 
 var _ = require('lodash');
-var async = require('async');
 var constants = require('../helpers/constants.js');
 var extend = require('extend');
 var fs = require('fs');
 var ip = require('ip');
 var OrderBy = require('../helpers/orderBy.js');
 var path = require('path');
 var Peer = require('../logic/peer.js');
-var PeerSweeper = require('../logic/peerSweeper.js');
+var pgp = require('pg-promise')(); // We also initialize that library here
```

Comment:
`We also initialize that library here` > `We also initialize library here`

We need to initialize?